servers/visual/rasterizer.h

```diff
@@ -1181,6 +1192,16 @@ class RasterizerCanvas {
 			return rect;
 		}
 
+		void remove_references(const RID &p_rid) {
+			for (int i = commands.size() - 1; i >= 0; i--) {
+				if (commands[i]->contains_reference(p_rid)) {
+					// Delete the command and pop the last.
+					memdelete(commands[i]);
+					commands.resize(commands.size() - 1);
```

Comment:
Just noticed a bug here, the command isn't always the last in the list so it needs an unordered remove I think. Just fixing.

Reply:
Fixed.